sdk/eventhub/tests.yml

```diff
@@ -5,27 +5,8 @@ stages:
     parameters:
       AllocateResourceGroup: false
       ServiceDirectory: eventhub
-      Matrix:
-        Linux_Python35:
-          Pool: azsdk-pool-mms-ubuntu-1804-general
-          OSVmImage: MMSUbuntu18.04
-          PythonVersion: '3.5'
-          CoverageArg: '--disablecov'
-        Linux_Python39:
-          Pool: azsdk-pool-mms-ubuntu-1804-general
-          OSVmImage: MMSUbuntu18.04
-          PythonVersion: '3.9'
-          CoverageArg: ''
-        Windows_Python27:
-          Pool: azsdk-pool-mms-win-2019-general
-          OSVmImage: MMS2019
-          PythonVersion: '2.7'
-          CoverageArg: '--disablecov'
-        MacOs_Python37:
-          Pool: Azure Pipelines
-          OSVmImage: 'macOS-10.15'
-          PythonVersion: '3.7'
-          CoverageArg: '--disablecov'
+      MatrixFilters:
+        - PythonVersion=^(?!pypy3).*
```

Comment:
This seems like a pretty common filter. That matrix entry seems like we might want to flip it to be opt-in instead of opt-out.

Reply:
There are three services excluding it, and eighteen using the default that includes it.